x-pack/plugins/lens/public/editor_frame_plugin/editor_frame/editor_frame.tsx

```diff
@@ -114,20 +171,22 @@ export function EditorFrame(props: EditorFrameProps) {
           />
         }
         workspacePanel={
-          <WorkspacePanel
-            activeDatasource={props.datasourceMap[state.datasource.activeId]}
-            activeVisualizationId={state.visualization.activeId}
-            datasourcePublicAPI={datasourcePublicAPI!}
-            datasourceState={state.datasource.state}
-            visualizationState={state.visualization.state}
-            visualizationMap={props.visualizationMap}
-            dispatch={dispatch}
-            ExpressionRenderer={props.ExpressionRenderer}
-          />
+          <WorkspacePanelWrapper title={state.title} dispatch={dispatch}>
```

Comment:
any reason this isn't part of WorkspacePanel?

Reply:
It's a different responsibility, as was evidenced by the broken tests when I (originally) made this change to WorkspacePanel. The WorkspacePanel is primarily responsible for managing the expression rendering and things like that. This is more of a basic wrapper around that, and will essentially manage the header of the WorkspacePanel. It might be worth making it WorkspacePanelHeader instead or something like that.

Reply:
Makes sense, thanks.